style_guide.md

```diff
@@ -1,26 +1,100 @@
-# BODS Style Guide - #draft
+# BODS Style Guide - #WIP
+
+This style guide is a draft to be discussed and not yet implemented across the BODS schema and documentation (as of May 2019, BODS v0.2). It can be used to guide editing and content writing.
+## Common Conventions
+
+### Readability 
+We try to write clear, concise documentation, using plain English where possible. 
+
+These tools can help with this:
+- [Hemingway editor](https://hemingwayapp.com/)
+- [Grammarly](https://www.grammarly.com/)
+- [Readability tools](https://www.webfx.com/tools/read-able/)
+
+### Spelling 
+Use English spelling:
+- 'organisation' not 'organization'
+- 'standardised' not 'standardized'
+- 'modelling' not 'modeling'
+
+### Word choice
+- use 'person' instead of using 'individual' as a noun
+- 'changelog' not 'change-log,' 'change log,' or 'changeLog'
+- 'codelist' not 'code-list,' 'code list' or 'codeList'
+- 'politically exposed' not 'politically-exposed'
+- 'free text' not 'free-text'
+- do not use constructions like “person(s)” or “the supplier (or suppliers)”. The plural is fine, like “persons.” *Should this also include e.g 'One or more addresses for this entity.' -> 'addresses for this entity.'*
+
+### Bulleted lists
+When using bulleted lists you should:
+- start with a lead in line, followed by a colon
+- use lower case at the start of each bullet
+- do not put semicolons or full stops at the end of bullets
+- do not put or/and at the end of bullets
+- try to have one sentence per bullet
+
+## Schema style guide
```

Comment:
We probably want to maintain the distinction between the Schema style guide and the BODS documentation style guide. The former will only relate to: schema property names and titles, property descriptions, codelist names, codelist item naming etc. The BODS documentation style guide will go beyond what you've noted in 'common conventions' and set out how to refer to data elements: objects, properties, codelist values etc. Can talk more about this tomorrow.